source/index.html

```diff
@@ -22,6 +22,12 @@ <h4>Comments</h4>
       </li>
     </ul>
 
+    <form id='new_comment'>
```

Comment:
Is there a way we could have done this without editting the HTML? Can we use jQuery to add this to the DOM?